src/main/java/org/opentripplanner/middleware/controllers/api/BugsnagController.java

```diff
@@ -56,17 +59,22 @@ public void bind(final SparkSwagger restApi) {
     /**
      * Get all Bugsnag events from Mongo and replace the project id with the project name and return
      */
-    private static List<EventSummary> getEventSummary(Request request, Response response) {
+    private static ResponseList<EventSummary> getEventSummary(Request req, Response res) {
         List<EventSummary> eventSummaries = new ArrayList<>();
         List<BugsnagEvent> events = bugsnagEvents.getAll();
+        int limit = Integer.parseInt(req.queryParamOrDefault("limit", "10"));
```

Comment:
Add the spark-swagger code to document the new query parameters `limit` and `page`. (This is an admin endpoint so this is for folks who develop against this endpoint).